pages/config-service-configuration/add-or-edit-chain.mdx

```diff
@@ -0,0 +1,195 @@
+# Add or edit chain
+
+Add (or edit) a blockchain network.
+
+## Usage
+
+You can add a new chain by visiting this address: [`http://localhost:8000/cfg/admin/chains/chain/add/`](http://localhost:8000/cfg/admin/chains/chain/add/)
+
+You can edit an existing chain by visiting this address: `http://localhost:8000/cfg/admin/chains/chain/{chain index}/change/` where `{chain index}` is the index of the chain you want to edit.
+
+## Parameters
+
+### `ChainId`
+
+The unique identifier for the chain.
+
+### `Relevance`
+
+The relative importance of this chain for your project. You can use this value between 0 and 100 to sort chains in the UI.
+
+### `Chain Name`
+
+The name of the chain, in plain letters.
+
+### `EIP-3770 short name`
+
+The short name of the chain, as defined in [EIP-3770](https://eips.ethereum.org/EIPS/eip-3770). This corresponds to the three letters that will be prepended to all addresses on this chain.
+
+### `Password`
+
+Opens a section to set a password for the user. You can set a new password or leave it blank to keep the current one.
+
+### `Description`
+
+A brief description of the chain.
+
+### `Chain logo URI`
+
+The URI of the chain's logo. This will be displayed in the UI.
+
+### `L2`
+
+Whether this chain is a Layer 2 chain. This has important consequences on the indexing ability of the chain. If this chain is an Ethereum Layer 2, not checking this box will increase the indexing time, hence possibly missing the correct indexing of some transactions.
```

Comment:
I don't believe the Transaction Service consumes the Config Service - this is only used client-side.